packages/bpk-component-map/src/BpkMapMarker.scss

```diff
@@ -0,0 +1,47 @@
+@import '~bpk-mixins';
+
+.bpk-map-marker {
+  display: flex;
+  width: $bpk-spacing-lg;
+  height: $bpk-spacing-lg;
+  justify-content: center;
+  align-items: center;
+  border: $bpk-border-size-xl solid $bpk-color-white;
+  border-radius: 50%;
+  background-color: $bpk-color-green-700;
+  color: $bpk-color-white;
+
+  @include bpk-box-shadow-lg;
+
+  &--large {
+    width: $bpk-spacing-xxl;
+    height: $bpk-spacing-xxl;
+  }
+
+  &--dynamic {
+    cursor: pointer;
+  }
+
+  &__wrapper {
+    position: relative;
+    display: flex;
+    padding-bottom: 5px; // stylelint-disable-line unit-blacklist
```

Comment:
I understand why you need to opt out of the rule here - could you add a quick, one-line comment to explain it's because of the arrow positioning.

Our convention for eslint is to do this every time we opt out, I'll change the convention to include stylelint too: https://github.com/Skyscanner/backpack/blob/master/decisions/eslint-ignores.md

Reply:
Sure, makes total sense 👍

Reply:
Added